code/src/components/NewTodo.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import uniqid from 'uniqid'
+
+import todos from '../reducers/todos'
+
+
+const Input = styled.input`
+  width:100%;
+  border: none;
+  outline:none;
+  background: none;
+  font-size: 18px;
+  color: #555;
+  padding:20px 10px 20px 5px;
```

Comment:
IMHO, it would look much better with bigger padding to make the content text start not in the shadowed part :)